charts/karpenter/values.yaml

```diff
@@ -18,3 +18,12 @@ webhook:
   tolerations: []
   affinity: {}
   image: "public.ecr.aws/karpenter/webhook:v0.4.2@sha256:2d004cd277209242d7e886308be419f86107811b4d0f1ed21c908b77e284202b"
+defaultProvisioner:
+  create: true
+  ttlSecondsAfterEmpty: 300
+  #ttlSecondsUntilExpired: 86400 
```

Comment:
Just wondering - do we want to leave this commented-out value here?

Reply:
I was going to leave it here to show that it is configurable since we don't have great helm docs now.